pages/index.tsx

```diff
@@ -31,7 +31,8 @@ const Home: NextPage = () => {
       <LeftSidebarLayout>
         <Container>
           <SectionIntro
-            backgroundImage="/resources/svg/bg-pools.svg"
+            backgroundImage="/resources/svg/bg-deals.svg"
+            backgroundPosition="100% 110%"
```

Comment:
Why 110%?

Reply:
Because the SVG is big and it was the way I found to adapt it to different screen sizes. If it's in `px`, some screen sizes don't fit well.